packages/sdk-react/src/components/payables/PayablesTable/Filters/Filters.tsx

```diff
@@ -17,7 +17,6 @@ import { Theme } from 'mui-styles';
 
 import { getRowToStatusTextMap } from '../../consts';
 import {
-  FILTER_TYPE_CREATED_AT,
```

Comment:
Since you are removing this filter type, this constant could be removed from constants, filter type, and the table data request
`packages/sdk-react/src/components/payables/PayablesTable/consts.ts`
`packages/sdk-react/src/components/payables/PayablesTable/types.ts`
`packages/sdk-react/src/components/payables/PayablesTable/PayablesTable.tsx`

Reply:
Good catch, let me adjust that